dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/ApplicationProcessor.java

```diff
@@ -89,6 +91,12 @@ public class ApplicationProcessor {
     @Reference
     private ValidationComponent validationComponent;
 
+    @Reference
+    private volatile List<OpenAPIVersionUsageChecker> versionUsageCheckers;
+
+    @Reference
+    private OpenAPIVersionConfig versionConfig;
```

Comment:
Need to double-check this service does not go away when the `<mpOpenAPI>` config element is added or removed.

Reply:
Verified that it does not.